src/Symfony/Component/Serializer/Tests/Normalizer/ConstraintViolationListNormalizerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Normalizer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\Normalizer\ConstraintViolationListNormalizer;
+use Symfony\Component\Validator\ConstraintViolation;
+use Symfony\Component\Validator\ConstraintViolationList;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class ConstraintViolationListNormalizerTest extends TestCase
+{
+    private $normalizer;
+
+    protected function setUp()
+    {
+        $this->normalizer = new ConstraintViolationListNormalizer();
+    }
+
+    public function testSupportsNormalization()
+    {
+        $this->assertTrue($this->normalizer->supportsNormalization(new ConstraintViolationList()));
+        $this->assertFalse($this->normalizer->supportsNormalization(new \stdClass()));
+    }
+
+    public function testNormalize()
+    {
+        $list = new ConstraintViolationList(array(
+            new ConstraintViolation('a', 'b', array(), 'c', 'd', 'e', null, 'f'),
+            new ConstraintViolation('1', '2', array(), '3', '4', '5', null, '6'),
+        ));
+
+        $expected = array(
+            'title' => 'An error occurred',
```

Comment:
RFC 7807 is unambiguous on this point:

> **[3.1](https://tools.ietf.org/html/rfc7807#section-3.1).  Members of a Problem Details Object**
>
>    A problem details object can have the following members:
>
>    - "type" (string) - ... When
>    this member is not present, its value is assumed to be
>    "about:blank".

> [4.2](https://tools.ietf.org/html/rfc7807#section-4.2).  Predefined Problem Types
>
>    This specification reserves the use of one URI as a problem type:
>
>    The "about:blank" URI [RFC6694], when used as a problem type,
>    indicates that the problem has no additional semantics beyond that of
>    the HTTP status code.
>
>    When "about:blank" is used, the title SHOULD be the same as the
>    recommended HTTP status phrase for that code (e.g., "Not Found" for
>    404, and so on), although it MAY be localized to suit client
>    preferences (expressed with the Accept-Language request header).
>
>    Please note that according to how the "type" member is defined
>    (Section 3.1), the "about:blank" URI is the default value for that
>    member.  Consequently, any problem details object not carrying an
>    explicit "type" member implicitly uses this URI.

Perhaps:

```json
{
  "type": "/validation-error",
  "title": "Validation failed",
  ...
}
```